azurelinuxagent/ga/policy/policy_engine.py

```diff
@@ -15,24 +15,51 @@
 # Requires Python 2.4+ and Openssl 1.0+
 #
 
+import json
+import os
 from azurelinuxagent.common import logger
 from azurelinuxagent.common.event import WALAEventOperation, add_event
 from azurelinuxagent.common import conf
 from azurelinuxagent.common.exception import AgentError
+from azurelinuxagent.common.protocol.extensions_goal_state_from_vm_settings import _CaseFoldedDict
+from azurelinuxagent.common.osutil.default import DefaultOSUtil
+
+
+# Customer-defined policy is expected to be located at this path.
+# If there is no file at this path, default policy will be used.
+_CUSTOM_POLICY_PATH = DefaultOSUtil().get_custom_policy_file_path()
```

Comment:
```suggestion
_CUSTOM_POLICY_PATH =  get_osutil().get_custom_policy_file_path()
```

The intention is that distros can customize the location of those files